client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -66,6 +75,24 @@ class ImageEditorToolbar extends Component {
 	onAspectOpen( event ) {
 		event.preventDefault();
 
+		const { imageMeetsMinimumDimensions, translate } = this.props;
+
+		if ( imageMeetsMinimumDimensions === false ) {
+			const noticeText = translate(
+				'To change aspect ratio, your image dimensions should be greater than %(width)px wide ' +
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?